crown-court-proceeding/src/main/resources/application.yaml

```diff
@@ -90,3 +90,10 @@ cloud-platform:
 crown-court-proceeding:
   security:
     issuer-uri: http://localhost:${server.port}
+
+maatApi:
+  oAuthEnabled: true
+  baseUrl: ${MAAT_API_BASE_URL}
+  assessments-domain: /api/internal/v1/assessment
+  ioj-appeal-endpoints:
+    find-url: ${maatApi.assessments-domain}/ioj-appeal/repId/{repId}/current-passed
```

Comment:
Change this to Query Param

Reply:
LCAM-322 is created to support query params handling in the webclient, also waiting on the outcome of GraphQL implementation spike.